markdown/developers/tutorials/dsl2_modules_tutorial.md

```diff
@@ -11,414 +11,298 @@ If you create a new module with the goal of contributing the code to *nf-core*,
 
-```bash
-echo $VERSION >${software}.version.txt
-```
+---
 
-Our final script will therefore look like this:
+## Writing a new module
 
-```nextflow
-// Import generic module functions
-include { initOptions; saveFiles; getSoftwareName } from './functions'
+Check out [Tutorial: Create a DSL2 Module](https://nf-co.re/developers/tutorials/dsl2_modules_tutorial) for how to create a new DSL2 Module
 
-params.options = [:]
-options        = initOptions(params.options)
+### New module workflow
 
-def VERSION = '1.3.0'
+We have implemented a number of commands in the `nf-core/tools` package to make it incredibly easy for you to create and contribute your own modules to nf-core/modules.
 
-/*
-unfortunately need to output the version manually
-as done for module
-https://github.com/nf-core/modules/blob/master/software/homer/annotatepeaks/main.nf
-because the solution adopted in iVar, i.e.
-echo \$(fgbio --version 2>&1) >${software}.version.txt
-for FGBIO still generates an error exit in Nextflow for some reasons
-*/
+1. Install the latest version of [`nf-core/tools`](https://github.com/nf-core/tools#installation) (`>=2.1`)
 
-process FGBIO_FASTQTOBAM {
-    tag "$meta.id"
-    label 'process_low'
-    publishDir "${params.outdir}",
-        mode: params.publish_dir_mode,
-        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), publish_id:meta.id) }
+    > ⚠️ [2021-11-26] please use `dev` version of tools
 
-    conda (params.enable_conda ? "bioconda::fgbio=1.3.0" : null)
-    if (workflow.containerEngine == 'singularity' && !params.singularity_pull_docker_container) {
-        container "https://depot.galaxyproject.org/singularity/fgbio:1.3.0--0"
-    } else {
-        container "quay.io/biocontainers/fgbio:1.3.0--0"
-    }
+2. Install [`Nextflow`](https://www.nextflow.io/docs/latest/getstarted.html#installation) (`>=21.04.0`)
+3. Install any of [`Docker`](https://docs.docker.com/engine/installation/), [`Singularity`](https://www.sylabs.io/guides/3.0/user-guide/) or [`Conda`](https://conda.io/miniconda.html)
+4. [Fork and clone the nf-core/modules repo locally](#uploading-to-nf-coremodules)
+5. Set up git on your computer by adding a new git remote of the main nf-core git repo called `upstream`
 
-    input:
-    tuple val(meta), path(reads)
-    val(read_structure)
+   ```bash
+   git remote add upstream https://github.com/nf-core/modules.git
+   ```
 
-    output:
-    tuple val(meta), path("*_umi_converted.bam"), emit: umibam
-    path "*.version.txt"          , emit: version
+   Make a new branch for your module and check it out
 
-    script:
-    def software = getSoftwareName(task.process)
-    def prefix   = options.suffix ? "${meta.id}${options.suffix}" : "${meta.id}"
+   ```bash
+   git checkout -b fastqc
+   ```
```

Comment:
not particularly attached to previous screenshots, but there's 2 ways of creating a branch and one is through GitHub itself on the browser. We are presenting only one of the 2 options here.
This is usually a personal preference and I believe a tutorial should give people the option to choose.